src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -6,13 +6,14 @@
 
 import org.apache.log4j.BasicConfigurator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.log4j.BasicConfigurator'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)